analyzers/src/SonarAnalyzer.CSharp/Rules/CommentedOutCode.cs

```diff
@@ -46,7 +46,7 @@ public sealed class CommentedOutCode : SonarDiagnosticAnalyzer
                     }
                 });
 
-        private static void CheckTrivias(SyntaxTreeAnalysisContext context, IEnumerable<SyntaxTrivia> trivias)
```

Comment:
clean as we code: trivia is already plural form, the `s` at the end, both in method name and in the parameter, should be removed
https://dictionary.cambridge.org/dictionary/english/trivia